app/controllers/reviews_controller.rb

```diff
@@ -47,6 +49,12 @@ def destroy
   end
 
   protected
+  def authorize_user
+    if !user_signed_in? &&
+       !(current_user.admin? || @review.user == current_user)
```

Comment:
Use 4 (not 3) spaces for indenting a condition in an `if` statement spanning multiple lines.